### PR TITLE
fix: more broken snippets

### PR DESCRIPTION
**Pull request by rflechtner**
## fixes https://github.com/KILTprotocol/ticket/issues/3112

Fixed indented code block that are no longer rendered correctly in Docusaurus 3 by making them TsJsBlocks/Snippets.
Wasn't sure why they were not already, and whether we want to show the whole code or just the function contents.
Also fixed one use of the TsJsSnippet that was broken for the js part.

The snippet component is so brittle is practically unusable though.

## How to test:

Look at the pages.

## Checklist:

- [x] I have verified that the code works
- [ ] I have verified that the code is easy to understand
  - [ ] If not, I have left a well-balanced amount of inline comments
- [ ] I have [left the code in a better state](https://deviq.com/principles/boy-scout-rule)
- [ ] I have documented the changes (where applicable)
    * Either PR or Ticket to update [the Docs](https://github.com/KILTprotocol/docs)
    * Link the PR/Ticket here

### CODE REVIEW

File: docs/develop/07_dApp/04_verifier.md [hunk 1]
```diff
@@ -34,22 +37,26 @@ In this guide the website requests an email address that is owned by the DID.
...
+</TsJsSnippet>
 
 After settled on a CType, the server can build the request for the visitor.
 Since we want to ensure that the presentation of the credential is fresh, the server first has to create a random challenge.
 The presentation must include this challenge and since it's random, the presentation must be created and signed from scratch.
 This ensures that it's not possible to record a presentation and just send this, pretending to be the owner of the DID.
 The challenge can be generated using the polkadot crypto utilities:
 
-<TsJsBlock>
+<CodeBlock language="js">
```
Comment (rflechtner) [inline docs/develop/07_dApp/04_verifier.md]: this was not even TS, so why have the switcheroo?

Comment (ntn-x2) [reply docs/develop/07_dApp/04_verifier.md]: What if this snippet changes to include some TS syntax in the future? Why can't we keep it as a `TsJsBlock`?

Comment (ntn-x2) [reply docs/develop/07_dApp/04_verifier.md]: Also I think it's more consistent if all code snippets show the Ts-JS switch.

**Review by ntn-x2 — APPROVED** [review]
LGTM! I would still keep the TS-JS switch for all code snippets and not assume that they will not diverge at some point between TS and JS version.

File: docs/develop/07_dApp/04_verifier.md (hunk 1 above)
Comment (rflechtner) [reply docs/develop/07_dApp/04_verifier.md]: Mhh you're, doesn't hurt and afaik you set the language once for all examples, right?

Comment (ntn-x2) [reply docs/develop/07_dApp/04_verifier.md]: Correct.

**Pull request merged**